docs/MapColonies/Raster/services/catalog/profile_v1_0.md

```diff
@@ -25,27 +25,80 @@ tags:
 | mc:productSubType | text | ProductSubType values |
 | mc:links | text | available links for current product CSW Links <br /> structure of links in the format ***name,description,protocol,url[^„,[^„,]]*** |
 | mc:description | text | the product description |
-| mc:creationDateUTC | date | the date (UTC time) when raw product was created <br/> supported format: **dd/mm/yyyy** |
+| mc:creationDateUTC | date | **(Deprecated - will be removed in the future)** the date (UTC time) when raw product was created <br/> supported format: **dd/mm/yyyy** |
```

Comment:
Deprications should also be written in red at the top of the page.
In addition, the deprication text in the description should be in red as well.
Third, it should be on a different line (<br/>).

Reply:
fixed